FirebaseCore/Sources/FIRApp.m

```diff
@@ -822,6 +822,7 @@ + (void)registerSwiftComponents {
   // Dictionary of class names that conform to `FIRLibrary` and their user agents. These should only
   // be SDKs that are written in Swift but still visible to ObjC.
   NSDictionary<NSString *, NSString *> *swiftComponents = @{
+    @"FIRSessions" : @"fire-sessions",
```

Comment:
@samedson, this answers the question in the PR description, right?

Reply:
It's kindof part of the question. I have this SessionProvider but it doesn't seem to do anything besides satisfy the constraint that the Component must be passed a protocol. We use the init method in the Sessions class to start the SDK, not the `sessions(for app)` method that's defined by the protocol.